templates/site/torrents/view.jet.html

```diff
@@ -271,6 +271,34 @@ <h1 style="text-align: center;" class="torrent-hr">{{Torrent.Name}}</h1>
 {{end}}
 {{ block footer_js()}}
 <script type="text/javascript" src="{{ URL.Parse("/js/modal.js") }}"></script>
+{{if Torrent.LastScrape.IsZero || formatDateRFC(Torrent.LastScrape) == "0001-01-01T00:00:00Z"}}
```

Comment:
both conditions are identical
rtfm https://godoc.org/time#Time.IsZero

Reply:
Torrent.LastScrape.IsZero constantly returns false, i even made an issue about it a month ago

Reply:
then use just the second condition?

Reply:
i plan to fix .IsZero but as of right now we're doing it that way, that double condition exists multiple time in base.jet.html already, doing it the same way everytime is better because it allows to fix everything at once
consistency

Reply:
¯\_(ツ)_/¯